apps/analysis_framework/tests/test_mutations.py

```diff
@@ -774,6 +774,73 @@ def _query_check(**kwargs):
         response = _query_check()['data']['analysisFramework']['analysisFrameworkMembershipBulk']
         self.assertMatchSnapshot(response, 'try 2')
 
+    def test_analysis_framework_clone(self):
+        query = '''
+            mutation MyMutation ($id: ID!, $input: AnalysisFrameworkCloneInputType!) {
+              __typename
+              analysisFramework (id: $id ) {
+                analysisFrameworkClone(data: $input) {
+                  ok
+                  errors
+                  result {
+                    id
+                    title
+                    description
+                    clonedFrom
+                  }
+                }
+              }
+            }
+        '''
+
+        member_user = UserFactory.create()
```

Comment:
We will need additional users here.

Basically,
- User with normal access to project  & AF <- Fail
- User with normal access to project & but not AF (Private) <- Fail

- User with admin access to project but not AF (Private) <- Fail
- User with admin access to project & AF <- Success


Let's add check that we don't create any additional AF or change project.af until it is set.

Reply:
@AdityaKhatri  For
> User with admin access to project & AF (Private with access) <- Success

Do we need to check the clone AF is also private?